src/main/java/racingcar/domain/game/RacingGame.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.domain.game;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import racingcar.domain.car.Car;
+import racingcar.domain.car.DefaultCar;
+import racingcar.domain.numbergenerator.DefaultRandomNumberGenerator;
+import racingcar.domain.numbergenerator.RandomNumberGenerator;
+import racingcar.view.RacingGameView;
+
+public class RacingGame implements Game {
+
+    private List<Car> cars;
+    private int attempts;
+
+
+    @Override
+    public void start() {
+        RacingGameView view = new RacingGameView(this);
+        initialize(view);
+        race(view);
+    }
+
+    private void race(RacingGameView view){
+        for (int i = 0; i < attempts; i++) {
+            moveCars();
+            view.printCurrentStatus();
```

Comment:
RacingGameView를 의존하지 말고, RacingGameView가 Car를 의존하게 하는 건 어떨까요?

```suggestion
            RacingGameView.printCurrentStatus(cars);
```